apps/src/code-studio/pd/application_dashboard/regional_partner_dropdown.jsx

```diff
@@ -26,13 +25,15 @@ export class RegionalPartnerDropdown extends React.Component {
     regionalPartners: PropTypes.arrayOf(PropTypes.shape({
       id: PropTypes.number,
       name: PropTypes.string
-    }))
+    })),
+    otherOptions: PropTypes.array,
```

Comment:
How about `additionalOptions`, since it's applied as a concatenation rather than substitution?